core-java-8/src/test/java/com/baeldung/java8/Java8MaxMinUnitTest.java

```diff
@@ -44,4 +46,29 @@ public void whenListIsOfPersonObjectThenMinCanBeDoneUsingCustomComparatorThrough
         assertEquals("Should be Alex", alex, minByAge);
     }
 
+    @Test
+    public void whenArrayIsOfIntegerThenMinCanBeDoneUsingIntegerComparator() {
+        int[] integers = new int[] { 20, 98, 12, 7, 35 };
+
+        int min = Arrays.stream(integers)
+            .min()
+            .getAsInt();
+
+        assertEquals(7, min);
+    }
+
+    @Test
+    public void whenArrayIsOfPersonObjectThenMaxCanBeDoneUsingCustomComparatorThroughLambda() {
```

Comment:
I edited the names of these unit tests in the article so they didn't bleed across the margin. Would you please update these names to match?